locales/en/settings.ftl

```diff
@@ -59,6 +59,26 @@ settings-deactivate-account-title = Deactivate account
 settings-deactivate-account-info-2 = You can deactivate { -product-short-name } by deleting your { -brand-mozilla-account }.
 settings-fxa-link-label-3 = Go to { -brand-mozilla-account } settings
 
+## Delete Monitor account
+
+settings-delete-monitor-free-account-title = Delete { -brand-monitor } account
+settings-delete-monitor-free-account-description = This will permanently delete your { -brand-monitor } account and turn off all notifications.
+settings-delete-monitor-free-account-cta-label = Delete account
+settings-delete-monitor-free-account-dialog-title = Your { -brand-monitor } account will be permanently deleted
+settings-delete-monitor-free-account-dialog-lead = All of your { -brand-monitor } account information will be deleted and we’ll no longer monitor for new data breaches. This will not delete your { -brand-mozilla } account.
+settings-delete-monitor-free-account-dialog-cta-label = Delete account
+settings-delete-monitor-free-account-dialog-cancel-button-label = Never mind, take me back
+settings-delete-monitor-plus-account-title = Delete { -brand-monitor } account
+settings-delete-monitor-plus-account-description = This will permanently delete your { -brand-monitor } account and immediately end your paid { -brand-monitor-plus } subscription.
+settings-delete-monitor-plus-account-cta-label = Delete account
+settings-delete-monitor-plus-account-dialog-title = Your { -brand-monitor } account will be permanently deleted
+settings-delete-monitor-plus-account-dialog-lead-p1 = All of your { -brand-monitor } account information will be deleted and we’ll no longer monitor for new data breaches or data broker exposures. This will not delete your { -brand-mozilla } account.
```

Comment:
Note to self, I should have caught this 😭 

`Mozilla account` is `{ -brand-mozilla-account }`, not a combination of Mozilla brand and account. Since I've already exposed these strings for translation, we should keep this in at this point.

Reply:
Oh yes, good point :/ Maybe a good addition to the linter too, to catch instances of `{ -brand-mozilla } account`? Theoretically that could run into false positives, but probably not in practice, and false negatives are the same as the status quo.